src/oabot/templates/change.html

```diff
@@ -46,6 +55,14 @@
         <ol>
 
         {%  for template_edit in proposed_edits %}
+            <div class="sherparomeo">
+                <b>Copyright guidance</b><br>
+                <div class="sherparomeonote">Are the authors of this citation permitted to archive the...</div>
+                Preprint: {{ circle_color( template_edit.policy.preprint ) }}<br>
+                Postprint: {{ circle_color( template_edit.policy.postprint ) }}<br>
+                Published: {{ circle_color( template_edit.policy.published ) }}<br>
+                <div class="sherparomeonote">Data from <i>Sherpa/Romeo</i></div>
+            </div>
```

Comment:
Nice! I did not know you could define macros in templates. What happens here if there is no policy stored? Shouldn't we first make sure it exists?

Reply:
Yes, good point.